package.json

```diff
@@ -17,7 +17,7 @@
     "postcompile": "tsc -p test; tsc -p typedocExamples",
     "prepack": "sf-build",
     "pretest": "sf-compile-test",
-    "test": "sf-test"
+    "test": "sf-test --require ts-node/register"
```

Comment:
@rorolopetegui This project defines a .mocharc.json, which has a require property that includes ts-node/register. Is sf-test not picking up that file?

Reply:
Sorry for late response. It was not picking up, the solution that i found for this was just by using the require flag.